assets/css/_route_properties_card.scss

```
@@ -0,0 +1,160 @@
+  display: flex;
+  flex-direction: row;
+  align-items: start;
+  padding: 0.5rem 0.5rem;
+  justify-content: space-between;
+  margin: 0;
+}
+
+.m-route-properties-card__route-title {
+  display: flex;
+  flex-direction: row;
+  gap: 0.5rem;
+  h2 {
+    font-size: 1.125rem;
+    line-height: 1;
+    margin: 0px;
+  }
+}
+
+.m-route-properties-card__route-description {
+  font-size: var(--font-size-s);
+  color: $color-gray-700;
+  font-weight: var(--font-weight-reg);
+}
+
+.m-route-properties-card__direction-picker {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: center;
+  padding: 0.25rem;
+  margin: 0rem 1rem 1rem 1rem;
+  gap: 4px;
+  background-color: $color-gray-100;
+  border-radius: 0.5rem;
+  height: 44px;
+  box-shadow: inset 0px 1px 2px rgba(0, 0, 0, 0.075);
+
+  input[type="radio"] {
+    display: none;
+  }
+}
+
+.m-route-properties-card__direction-picker .direction-button {
+  display: flex;
+  align-items: center;
+  flex: 1;
+  height: 36px;
+  cursor: pointer;
+  color: $color-gray-600;
+
+  label {
+    flex: 1;
+    font-size: 1rem;
+    text-align: center;
+    cursor: pointer;
+  }
+
+  &--selected {
+    color: $color-gray-900;
+    background-color: $white;
+    font-weight: var(--font-weight-semi);
+    border: 1px solid $color-gray-500;
+    border-radius: 0.5rem;
+    box-shadow: 0px 2px 4px rgba(0, 0, 0, 0.075);
+  }
+}
+
+.m-route-properties-card__summary {
+  font-size: var(--font-size-sm);
+  text-decoration: underline;
+  color: $color-eggplant-700;
+  padding: 0rem 1rem;
+  &--open {
+    font-weight: var(--font-weight-semi);
+  }
+}
+
+.m-route-properties-card__details-section.variant-picker fieldset {
+  border: none;
+  padding: 0rem;
+  margin: 0rem;
+}
+
+.m-route-properties-card__variant-option {
+  input[type="radio"] {
+    display: none;
+  }
+
+  label {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 1rem;
+    font-size: var(--font-size-s);
+    padding: 0.5rem 1rem;
+  }
+
+  &:hover {
+    background-color: $color-eggplant-100;
+  }
+
+  & > * {
+    cursor: pointer;
+  }
+
+  .variant-check {
+    visibility: hidden;
+    vertical-align: middle;
+    fill: $color-eggplant-800;
+    width: 16px;
+    height: 16px;
+  }
+}
+
+.m-route-properties-card__variant-option--selected {
+  background-color: $color-eggplant-100;
+  color: $color-eggplant-800;
+
+  .m-route-properties-card__variant-option-description {
+    color: $color-eggplant-700;
+  }
+  .variant-check {
+    visibility: visible;
+  }
+}
+
+.m-route-properties-card__variant-option-name {
+  font-weight: var(--font-weight-semi);
+}
+
+.m-route-properties-card__variant-option-description {
+  font-size: var(--font-size-xs);
+  color: $color-gray-700;
+}
+
+.m-route-properties-card hr {
+  height: 1px;
+  background-color: $color-gray-400;
+  border: none;
+}
+
+.m-route-properties-card__details-section.variant-stop-list {
+  max-height: 490px;
+  overflow: scroll;
```

I get weird scrollbar artifacts on firefox and chrome around the outbound stops, setting to auto seems to ensure that scroll is used for overflow, but doesn't show scrollbars unless an overflow happens.


```suggestion
  overflow: auto;
```

---

Thanks for the suggestion! I also accidentally removed the `ol` selector here - we only want the scroll to affect the list of stops itself, not the summary too. Will add that back!